### ISSUE TITLE
Is it possible to add a simple turn on/off torch feature?

### ISSUE THREAD

**Issue by tgomezc** [basst314/ngx-webcam#64]
While the library is very complete and easy to use i think this is a good feature that is missing.

**Comment by Farry1** [comment]
I am also very interested in this feature. Did you come up with something already?

**Comment by carlosabud** [comment]
Same here. Does anybody know anything about this feature?

**Comment by basst314** [comment]
Thanks for pushing for this. While the webcam component itself can be "disabled" by using `ng-if` and conditionally (not) rendering it, the component does not currently have a dedicated on/off switch.

I will put that on my todo list for future versions.

Does conditional rendering work for you as a workaround?

**Comment by carlosabud** [comment]
I think we are referring to different things. 

We are asking (or maybe requesting) for a camera torch or camera flash functionality.

**Comment by basst314** [comment]
Hi @carlosabud,

Thanks for clarifying! That sounds like an exciting feature! Will do some research and get back to you all here.

Cheers!

**Comment by sagarwasule** [comment]
Any solution for this feature ?

**Comment by maxulm** [comment]
Also interested in such a feature!

**Comment by LocoUnPocoNadaMas** [comment]
#169 I have this for now it's not a solution but it may help, it needs love (time to learn and write a lot of code).

**Comment by sophieelsa** [comment]
getPhotoCapabilities method from ImageCapture allow to manage torch on device phone. It works on my side

**Comment by LocoUnPocoNadaMas** [comment]
yes, after search for a while I found this [image-capture.d.ts](https://github.com/GoogleChromeLabs/snapshot/blob/master/typings/image-capture.d.ts)

**Comment by ganholete** [comment]
Any solution? How can I use it? It would be beautiful
